modules/gojira-core/src/main/java/com/flipkart/gojira/core/NoneHttpFilterHandler.java

```diff
@@ -37,11 +37,12 @@
    *
    * @param request wrapped original http request as a {@link
    *     HttpFilter.CustomHttpServletRequestWrapper} object
+   * @param requestMode this is the mode of execution of gojira at a request level
    * @return true if {@link javax.servlet.FilterChain#doFilter(ServletRequest, ServletResponse)}
    *     needs to be called, else false.
    */
   @Override
-  public boolean preFilter(HttpFilter.CustomHttpServletRequestWrapper request) {
+  public boolean preFilter(HttpFilter.CustomHttpServletRequestWrapper request, Mode requestMode) {
```

Comment:
Is the requestMode variable needed? The implementation is specific to Mode.NONE. We can remove this variable.

Reply:
Done